pagopa/io-app
# feat: [IOBP-201,IOBP-284] New wallet success outcome handling

## ⚠️ This PR depends on #5137 
## Short description
This PR adds an handling for the new wallet onboarding outcome managing also the wallet id created in order to navigate to the detail page as soon as the onboarding finish.

## List of changes proposed in this pull request
- Handled the `idWallet` for successful outcome;
- Refactor of the generic error page for failure/error outcome;
- Added a navigation to the onboarded card detail when the outcome is successful (currently it navigates only to the screen but must be handled into the detail screen page the new wallet implementation API, so it will be handled in another PR [[IOBP-373]](https://pagopa.atlassian.net/browse/IOBP-373))

## How to test
- Open New wallet playground section from Profile tab;
- Open Onboarding playground
- Go forward until a webview is present, choose the desired outcome and check that every outcome is handled correctly

## Preview

https://github.com/pagopa/io-app/assets/34343582/fb2bf196-15e0-4098-b33a-783a7e648912



[IOBP-373]: https://pagopa.atlassian.net/browse/IOBP-373?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
